src/System.ValueTuple/src/System.ValueTuple.csproj

```diff
@@ -1,58 +1,41 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
-  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
   <PropertyGroup>
-    <!-- rev'ed to 4.0.1.0 even though 4.0.0.0 never shipped so that we can drop pre-release down to beta -->
-    <ProjectGuid>{4C2655DB-BD9E-4C86-83A6-744ECDDBDF29}</ProjectGuid>
-    <DocumentationFile>$(OutputPath)$(MSBuildProjectName).xml</DocumentationFile>
-    <IsPartialFacadeAssembly Condition="'$(TargetGroup)' == 'netcoreapp' OR '$(TargetGroup)' == 'uap' OR '$(TargetGroup)' == 'net47'">true</IsPartialFacadeAssembly>
-    <RunApiCompat Condition="'$(TargetGroup)' != 'netcoreapp' AND '$(TargetGroup)' != 'uap'">false</RunApiCompat>
-    <!-- We've explicitly authored a reference facade so we do not need to use the desktop implementation as ref -->
-    <PackageDesktopAsRef>false</PackageDesktopAsRef>
+    <TargetFrameworks>netstandard2.0;$(NetFrameworkMinimum);net47;netcoreapp2.0</TargetFrameworks>
+    <AssemblyKey>Open</AssemblyKey>
+    <IsPlaceholderTargetFramework
+      Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$ (TargetFramework)')) == '.NETCoreApp'
+                 or ($([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'netstandard2.0'))
+                     and '$(TargetFramework)' != '$(NetFrameworkMinimum)')">true</IsPlaceholderTargetFramework>
```

Comment:
I would make this condition simpler: `'$(TargetFramework)' != 'net462'`. Use the hardcoded value here so that when NetFrameworkMinimum changes to net47 (in Jan 2027), the project automatically does the right thing.